pandas/tests/groupby/test_nth.py

```diff
@@ -530,3 +530,23 @@ def test_nth_nan_in_grouper(dropna):
     )
 
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("method", ["first", "last"])
```

Comment:
can you also test ```.nth(0)``` and ```.nth(-1)``` which are the same results (except the nth -1 will have the null as the result

Reply:
Added a test for nth with nulls; I gave it its own test rather than putting it with first / last to avoid too much awkward if / else branching inside the test